### PR TITLE
Fix reindex page

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1097464

### CODE REVIEW

**Comment by huangp** [comment]
:+1:

File: zanata-war/src/main/webapp/WEB-INF/layout/language/language_list.xhtml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+          id="more-actions">
+          <a class="dropdown__toggle js-dropdown__toggle" href="#"
+            title="#{msgs['jsf.tooltip.MoreActions']}"><i
+            class="i i--ellipsis"></i></a>
+          <ul class="dropdown__content js-dropdown__content" role="content"
+            aria-labelledby="dropdownContent">
+            <li>
+              <a href="#" class="i__item--right"
+                data-toggle="modal" onclick="return false;"
+                data-target="#newLanguageDialog">
```
Comment (djansen-redhat) [inline zanata-war/src/main/webapp/WEB-INF/layout/language/language_list.xhtml]: Visible by normal user

**Comment by djansen-redhat** [comment]
:x: Add new language visible to normal user (unusable)
:?: Admin, normal page - same page, different url?

**Comment by aeng** [comment]
The only difference of admin language page and normal language is the heading. The content should be the same.

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/6603f7cb6bd0bf5afed63c3a03b721cfd073ed48

**Pull request closed without merging**